src/tigerbeetle/inspect.zig

```diff
@@ -25,7 +25,7 @@ const is_composite_key = vsr.lsm.composite_key.is_composite_key;
 pub fn main(allocator: std.mem.Allocator, cli_args: *const cli.Command.Inspect) !void {
     var stdout_buffer = std.io.bufferedWriter(std.io.getStdOut().writer());
 
-    const stdout = stdout_buffer.writer();
+    const stdout = stdout_buffer.writer().any();
```

Comment:
:thinking: does this actually work as intended? I think we'll need one extra temporary here: `.writer()` stores the pointer to `stdout_buffer`, but `.any()` stores the pointer to `.writer()` (so, two start pointer). But `.writer` is a temporary, and it should die after the expression is  evaluated.

Reply:
Weirdly it does work, but I will update it b/c I agree that it shouldn't...